src/Compilers/CSharp/Portable/Binder/Semantics/Operators/UnaryOperatorSignature.cs

```diff
@@ -27,5 +26,20 @@ public override string ToString()
         {
             return $"kind: {this.Kind} operand: {this.OperandType} return: {this.ReturnType}";
         }
+
+        public RefKind RefKind
+        {
+            get
+            {
+                Debug.Assert(
+                    Method != null &&
```

Comment:
>Method != null && [](start = 20, length = 17)

It is not clear why this is a safe assumption to make. It looks like ```Method``` is optional.

Reply:
Yes. Adjusted based on other tests.

---
In reply to: [155019741](https://github.com/dotnet/roslyn/pull/23122#discussion_r155019741) [](ancestors = 155019741)